src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1419,11 +1419,17 @@ def _configure_security_policy_storage_params(arg_ctx):
     with self.argument_context('sql midb restore') as c:
         create_args_for_complex_type(
             c, 'parameters', ManagedDatabase, [
+                'deletion_date',
                 'target_managed_database_name',
                 'target_managed_instance_name',
                 'restore_point_in_time'
             ])
 
+        c.argument('deletion_date',
+                   options_list=['--deletion-date'],
```

Comment:
Can you align this with `az sql db restore`, which has `--deleted-time`? 

https://docs.microsoft.com/en-us/cli/azure/sql/db?view=azure-cli-latest#az-sql-db-restore